diffpy/pdfmorph/pdfplot.py

```diff
@@ -217,17 +217,17 @@ def truncatePDFs(r, gr, rmin = None, rmax = None):
 
 def _configure():
     """Configure look and feel."""
-    import pylab
-    pylab.rc("font", size = 40)
-    pylab.rc("axes", linewidth = 2, labelsize = 30)
-    pylab.rc("xtick", labelsize = 25)
-    pylab.rc("xtick.major", size = 7)
-    pylab.rc("xtick.minor", size = 3)
-    pylab.rc("ytick", labelsize = 25)
-    pylab.rc("ytick.major", size = 7)
-    pylab.rc("ytick.minor", size = 3)
-    pylab.rc("legend", fontsize = 18)
-    pylab.rc("lines", markeredgewidth = 2) # thicker axes and symbols
+    import matplotlib.pyplot as plt
+    plt.rc("font", size = 40)
```

Comment:
Should these be put into a style sheet?

Reply:
can we ship our group style-sheet in the package?  or package it and make it a dependency?  Then we can start imposing our style on the whole community ;)

Reply:
I'm not quite sure how to do that, but I could learn!

Reply:
So my suggestion would be that @RomanMatthew you create a new issue which captures the intent of this conversation, then we can resolve this thread  and move towards merging and closing this PR, then that additional enhancement can be worked on next.

Reply:
I think we can ship a style sheet with the package

Reply:
Let's ship the Billinge-group style sheet then, for now.   We really need an issue on ``billingegroup/`` somewhere to visit how we will do group style-sheets in general.  @RomanMatthew, perhaps you could post that issue (ask Tim or someone), but we won't work on it just yet.  Thx

Reply:
We have the option of providing just a URL but I think we should ship a hard copy, that way we can let users use it even if they have no internet.

Reply:
matplotlib can take a remote URL of a style.

https://matplotlib.org/3.1.0/api/style_api.html

we can expand the discussion in a separate issue later.

Reply:
Sorry, but where's billingegroup/ ? I was going to open an issue on diffpy, but maybe not if it should go somewhere else

Reply:
Just opened an issue on the page you linked